utilities/jfr/jenkinsfile-runner.inc

```diff
@@ -0,0 +1,17 @@
+# function to run a docker image
+# $1: tag of the docker image
+# $2: Path to Jenkinsfile
+run_jfr_docker_image() {
+
+	if [ "$#" -gt 1 ]
+	then
+		docker run --rm -v $2:/workspace/Jenkinsfile $1
```

Comment:
Ideally it needs to support passing parameters like Java Options. Maybe not in the first revision

Reply:
What do you mean? Using `-e ${JAVA_OPTION}`?
I agree maybe a follow-up is needed.

Reply:
I was thinking in using the same ticket since some parts are still pending (hooks for example), but I'm happy to open a new one if you think it would be useful.